components/pkg-export-kubernetes/src/bind.rs

```diff
@@ -29,7 +29,7 @@ impl FromStr for Bind {
 
     fn from_str(bind_spec: &str) -> Result<Self, Error> {
         let split: Vec<&str> = bind_spec.split(":").collect();
-        if split.len() != 3 {
+        if split.len() != 3 || split[0] == "" || split[1] == "" || split[2] == "" {
```

Comment:
Nice catch.

Reply:
thanks, i caught this when I added a testcase for a later PR. :)